src/ostorlab/serve_app/oxo.py

```diff
@@ -690,6 +690,8 @@ def mutate(
 
                 with models.Database() as session:
                     created_scan.agent_group_id = scan.agent_group_id
+                    session.add(created_scan)
```

Comment:
Why?

Reply:
The scan was never updated with the agent_group_id